src/main/java/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package strategy.impl;
+
+import strategy.OperationHandler;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the redundant empty line.